packages/base/src/index.mjs

```diff
@@ -4,418 +4,422 @@ import js from '@eslint/js';
 import importX from 'eslint-plugin-import-x';
 import jsdoc from 'eslint-plugin-jsdoc';
 import prettier from 'eslint-plugin-prettier/recommended';
-// @ts-expect-error - `eslint-plugin-promise` doesn't have TypeScript types.
+// @ts-ignore - `eslint-plugin-promise` doesn't have TypeScript types.
 import promise from 'eslint-plugin-promise';
 import globals from 'globals';
 import { createRequire } from 'module';
 
+import { createConfig } from './utils.mjs';
+
 // TODO: Use import attributes when ESLint supports them.
 const customRequire = createRequire(import.meta.url);
 const environmentRules = customRequire('./environment.json');
 
 /**
  * @type {import('eslint').Linter.Config[]}
  */
-const rules = [
```

Comment:
GitHub shows this as a lot of changes, but essentially it just wraps the entire config in `createConfig`. As you can tell by the rules snapshot, the actual config hasn't changed.